app/routes/ranger.$.tsx

```diff
@@ -24,8 +24,7 @@ function handleRedirects(context: Parameters<LoaderFunction>[0]) {
   reactRangerV1List.forEach((item) => {
     if (url.pathname.startsWith(`/ranger/v1/${item.from}`)) {
       throw redirect(
-        `/ranger/v1/${item.to}?from=reactRangerV1&original=https://react-ranger-v1.tanstack.com/${item.from}`,
-        301
+        `/ranger/v1/${item.to}?from=reactRangerV1&original=https://react-ranger-v1.tanstack.com/${item.from}`
```

Comment:
The latest version is `v0` according to `app/projects/ranger.tsx`, so shouldn't we redirect users to the `v0` docs? 🤔 (It does sound weird to go from v1 to v0, I admit that...)